### PR TITLE
Update User guide screenshots for Operate

### PR DESCRIPTION
**Pull request by huygur**
- Removed ...dark.png files because they are not used
- Removed 'light' extension from ...light.png files
- updated operate screenshots in the user guide
- Did some renamings because of changes in the app (for example, Instances tab is renamed to Processes and related places are updated in the document)

### CODE REVIEW

File: docs/components/operate/index.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ dark: useBaseUrl('img/operate-introduction_dark.png'),
 
 In addition to providing visibility into active and completed process instances, Operate also makes it possible to carry out key operations such as resolving [incidents](./userguide/resolve-incidents-update-variables.md), and updating process instance variables.
 
-[//]:# (What is the link to fix here?)
+[//]: # "What is the link to fix here?"
```
Comment (akeller) [inline docs/components/operate/index.md:23]: This comment should actually be addressed. @christinaausley can you follow up here?

**Comment by akeller** [comment]
I'll merge #755 first since the changes are so small and then we can review and merge this in. Thank you!

File: docs/components/operate/index.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/operate/index.md]: Removed this in my latest commit.

**Review by christinaausley — APPROVED** [review]
No technical adjustment on my end for this one.

**Pull request merged**